roles/base/tasks/main.yml

```diff
@@ -54,16 +76,24 @@
   copy:
     dest: /etc/profile.d/00golang.sh
     content: "export PATH=/opt/golang/bin:/usr/local/go/bin:$PATH\nexport GOPATH=/opt/golang"
+- name: remove older docker
```

Comment:
Can you provide specifics of how this can be achieved? If so, I can implement it, perhaps as a follow on.